src/eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -669,17 +669,25 @@ def fit_channel(meta, time, flux, chan, flux_err, eventlabel, params,
             ExpRampModel = dm.ExpRampModel
         else:
             ExpRampModel = m.ExpRampModel
-        t_ramp = ExpRampModel(parameters=params, name='ramp', fmt='r--',
-                              log=log, time=time, time_units=time_units,
-                              freenames=freenames,
-                              longparamlist=lc_model.longparamlist,
-                              nchannel=chanrng,
-                              nchannel_fitted=nchannel_fitted,
-                              fitted_channels=fitted_channels,
-                              paramtitles=paramtitles,
-                              multwhite=lc_model.multwhite,
-                              nints=lc_model.nints)
```

Comment:
These removed lines will need to be added back (as they're important for `multwhite` fits) and presumably need to be added to the `HSTRampModel` as well